alchemiscale/interface/api.py

```diff
@@ -640,10 +640,7 @@ def get_transformation_results(
     sk = ScopedKey.from_str(transformation_scoped_key)
     validate_scopes(sk.scope, token)
 
-    # get all ProtocolDAGResultRefs for the given transformation's results
-    refs: List[ProtocolDAGResultRef] = n4js.get_transformation_results(sk)
-
-    return [i.to_dict() for i in refs]
+    return [str(sk) for sk in n4js.get_transformation_results(sk)]
```

Comment:
just a query, because I remember discussing this at some point, is the pattern to return `str(sk)` or `sk` objects in general.

Reply:
This is for simpler JSON-serializability of `ScopedKey`s, since they can be fully-represented as strings. We could leave them as-is since they're just `pydantic` models, but they'd just have to processed as dicts instead client-side. Ends up being marginally easier this way IMHO.